src/firefly/js/charts/ChartUtil.js

```diff
@@ -668,7 +668,7 @@ export function applyDefaults(chartData={}, resetColor = true) {
     const noXYAxis = Boolean(!nonPieChart);
 
     const defaultLayout = {
-        hovermode: 'closest',
+        hovermode: 'compare',
```

Comment:
The change should apply only to histogram. We still want 'closest' for scatter. Otherwise we get double/triple tooltips for highlighted/selected points in scatter.